store/src/java/com/zimbra/cs/imap/ImapHandler.java

```diff
@@ -515,6 +520,11 @@ boolean executeRequest(ImapRequest req) throws IOException, ImapException {
                     }
                     checkEOF(tag, req);
                     return isProxied ? imapProxy.proxy(req) : doFETCH(tag, sequence, attributes, parts, byUID, modseq);
+                } else if (command.equals("FLUSHCACHE")) {
```

Comment:
RFC specifically includes this in the ABNF (ref: [https://tools.ietf.org/search/rfc3501#page-91](https://tools.ietf.org/search/rfc3501#page-91)):

`x-command       = "X" atom <experimental command arguments>`

FWIW, Gmail seems to use X-GM-... these days (ref: [https://developers.google.com/gmail/imap/imap-extensions](https://developers.google.com/gmail/imap/imap-extensions)).